megamek/src/megamek/client/ui/swing/lobby/ChatLounge.java

```diff
@@ -2715,16 +2719,33 @@ public void mouseReleased(MouseEvent e) {
             }
         }
 
-        /** Shows the right-click menu on the mek table */
+        @Override
+        public void mouseMoved(MouseEvent e) {
+            if (popup == null) return;
+           Point p = e.getLocationOnScreen();
+           if (!popup.contains(p))
+           {
+               closePopup();
+           }
+        }
+
+        private void closePopup()
+        {
+            if (popup == null) return;;
```

Comment:
Not sure we need double semicolons here.

Reply:
Also, I think it's an un-spoken standard that we put curly braces around even one-liner interior code blocks.

Reply:
It's part of our code style guide @NickAragua, at https://github.com/MegaMek/megamek/wiki/MegaMek-Coding-Style-Guide